ceph.spec.in

```diff
@@ -152,7 +152,8 @@ BuildRequires:	fuse-devel
 # enough.
 BuildRequires:	devtoolset-8-gcc-c++ >= 8.3.1-3.1
 %else
-BuildRequires:	gcc-c++
+# at least gcc9 is needed. If not available, use gcc (this requires rpm>=4.13)
```

Comment:
Maybe slightly rephrase: "at least gcc9 is needed for some build configurations, like -DWITH_SEASTAR=ON" ?

Reply:
adjusted. better?

Reply:
assuming you are happy. resoloving conversation.